app/javascript/packs/articlePage.jsx

```diff
@@ -48,7 +48,6 @@ const userDataIntervalID = setInterval(async () => {
           onUnsubscribe={subscriptionRequestHandler}
         />,
         root,
-        root.firstElementChild,
```

Comment:
This creeped in from a rebase I imagine. This is no longer required in Preact 10.x +